src/ted2zim/scraper.py

```diff
@@ -77,12 +87,64 @@ def __init__(
         self.languages = (
             [] if languages is None else [lang.strip() for lang in languages.split(",")]
         )
+
+        def get_iso_639_3_language(lang: str) -> str | None:
+            """Helper function to safely get ISO-639-3 code from input language"""
+            lang_info = get_language_details(lang, failsafe=True)
+            if lang_info:
+                return lang_info["iso-639-3"]
+            else:
+                logger.warning(
+                    f"Failed to get iso-639-3 language info for {lang}. "
+                    "This value will be missing in ZIM Language metadata."
+                )
+                return None
+
+        def sort_languages_hack(languages: set[str]) -> list[str]:
+            """This is a temporary hack to sort languages by importance in the ZIM
+
+            For now, if eng is among the list, we assume it is the most important
+            language. Otherwise list is kept as-is
+            """
+            if "eng" in languages:
```

Comment:
```suggestion
            languages.sort(key=lambda x: -1 if x == "eng" else 0)
```
And you don't have to care about it being there or not